x-pack/plugins/alerting/server/application/rule/methods/bulk_enable/bulk_enable_rules.ts

```diff
@@ -364,13 +357,15 @@ const bulkEnableRulesWithOCC = async (
           name: ruleNameToRuleIdMapping[rule.id] ?? 'n/a',
         },
       });
+    } else {
+      rules.push(rule);
     }
   });
   return {
     errors,
     // TODO: delete the casting when we do versioning of bulk disable api
     rules: rules as Array<SavedObjectsBulkUpdateObject<RuleAttributes>>,
-    accListSpecificForBulkOperation: [taskIdsToEnable],
+    accListSpecificForBulkOperation: [taskIdsFailedToBeEnabled],
```

Comment:
Seems weird that we'd change the result here to be the tasks that FAILED to get enabled, vs what was originally return, the task ids to enable.  But I haven't quite figured out what this field is used for in the first place, so maybe it's right ...